src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas3/DomainEventServiceTest.kt

```diff
@@ -71,23 +90,12 @@ class DomainEventServiceTest {
 
   private val detailUrl = "http://example.com/123"
 
-  private val domainEventService = buildService(emitDomainEventsEnabled = EventType.entries)
-  private val domainEventServiceEmittingDisabled = buildService(emitDomainEventsEnabled = listOf())
```

Comment:
this is a really nice pattern, i really dislike having to build multiple services to test config variations 👍